sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/ModelsRepositoryClientBuilder.java

```diff
@@ -157,31 +150,25 @@ public ModelsRepotioryClient buildClient() {
      * @return the created asynchronous ModelsRepositoryAsyncClient
      */
     public ModelsRepositoryAsyncClient buildAsyncClient() {
-        Objects.requireNonNull(endpoint, "'endpoint' cannot be null");
-
         Configuration buildConfiguration = this.configuration;
-        if (buildConfiguration == null)
```

Comment:
I don't see what was wrong with this format 😂

Reply:
:D